### PR TITLE
Cleaning up the Geolocation control.

### PR DESCRIPTION
**Pull request by mhaertwig**

### CODE REVIEW

**Comment by mhaertwig** [comment]
Ahaha.
The button needs to be removed though.

File: src/components/Geolocation/Geolocation.js [hunk 1]
```diff
@@ -60,8 +53,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-  className: 'tm-geolocation',
-  classNameActive: 'blink',
+  className: 'rs-geolocation',
   title: undefined,
```
Comment (oterral) [inline src/components/Geolocation/Geolocation.js]: title and className can be remove using spread props int eh render method.

Comment (friedjoff) [reply src/components/Geolocation/Geolocation.js]: To support the `active` class I kept the className prop. Also this improves documentation so we should think about doing this always.

File: src/components/Geolocation/Geolocation.js [hunk 2]
```diff
@@ -220,17 +206,25 @@ class Geolocation extends PureComponent {
...
     const { active } = this.state;
 
     return (
-      <Button
-        className={`${className} ${active ? classNameActive : ''}`}
-        title={title}
+      <div
+        role="button"
+        tabIndex="0"
+        className={`${className} ${active ? 'rs-blink' : ''}`}
```
Comment (oterral) [inline src/components/Geolocation/Geolocation.js]: @friedjoff  maybe or more generic class like ` rs-active` is more suitable.

File: src/components/Geolocation/Geolocation.js (hunk 1 above)
Comment (oterral) [reply src/components/Geolocation/Geolocation.js]: MAybe  you 're rigth. Can you add the classNme  for all the other components? thx

**Comment by friedjoff** [comment]
done

**Pull request merged**